### PR TITLE
Fixed the bug of outdated function in eavesdropping on channel section

### PR DESCRIPTION
**Pull request by pritamsinha2304**
Issue #79
Replaced the outdated sniffing function

### CODE REVIEW

File: examples/Eavesdropping on channels/Eavesdropping.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+from qunetsim.components import Host
+from qunetsim.components import Network
+from qunetsim.objects import Message
+from qunetsim.objects import Qubit
+from qunetsim.objects import Logger
+
+Logger.DISABLED = True
+
+amount_transmit = 5
```
Comment (stephendiadamo) [inline examples/Eavesdropping on channels/Eavesdropping.py]: Constants should be all upper case.

File: examples/Eavesdropping on channels/Eavesdropping.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    host_alice.start()
+
+    host_bob = Host('Bob')
+    host_bob.add_connection('Alice')
+    host_bob.add_connection('Eve')
+    host_bob.start()
+
+    host_eve = Host('Eve')
+    host_eve.add_connection('Bob')
+    host_eve.delay = 0.2
```
Comment (stephendiadamo) [inline examples/Eavesdropping on channels/Eavesdropping.py]: Delay probably unneeded.

**Review by stephendiadamo — CHANGES_REQUESTED** [review]

File: examples/Eavesdropping on channels/Eavesdropping.py (hunk 2 above)
Comment (stephendiadamo) [reply examples/Eavesdropping on channels/Eavesdropping.py:64]: Just small changes. Also could you change the file path to be camel case?

**Comment by stephendiadamo** [comment]
Just small changes. Also could you change the file path to be pothole case?

**Comment by pritamsinha2304** [comment]
I am sorry, if I mess up something while commit, merge etc., because this is my first open source contribution and also with Github

**Comment by pritamsinha2304** [comment]
> Just small changes. Also could you change the file path to be pothole case?

Can you tell what do you mean by "pothole case" ? I don't get it

**Comment by stephendiadamo** [comment]
> pothole case

It just means `something_like_this`. All lowercase, spaces are `_`.

**Comment by stephendiadamo** [comment]
No problem, you did everything fine. Just small inconsistencies with the
project style that I'd like you to change before I merge.

Thanks,
Stephen

**Comment by pritamsinha2304** [comment]
> > pothole case
> 
> It just means `something_like_this`. All lowercase, spaces are `_`.

I changed it. I think I did it correctly

**Review by stephendiadamo — CHANGES_REQUESTED** [review]
Sorry, very minor thing but can you change the filename to be all lowercase?

**Comment by pritamsinha2304** [comment]
> Sorry, very minor thing but can you change the filename to be all lowercase?

No problem

**Review by stephendiadamo — APPROVED** [review]

**Pull request merged**